docs/self-hosted/installation/manual.md

```diff
@@ -23,7 +23,7 @@ We recommend aliasing the `start` script to Directus' start for easier deploymen
 {
 	...
 	"scripts": {
-		"start": "directus start"
+		"start": "directus start",
```

Comment:
This trailing comma makes the json invalid if this were copy pasted into package.json

```suggestion
		"start": "directus start"
```

Reply:
thanks bud, didn't see that before haha